### PR TITLE
chore: npm version bump + release script

### PR DESCRIPTION
**Pull request by SwenSchaeferjohann**

### CODE REVIEW

File: .npmrc.example [hunk 1]
```diff
@@ -0,0 +1 @@
+//registry.npmjs.org/:_authToken=YOUR_AUTH_TOKEN
```
Comment (sergeytimoshin) [inline .npmrc.example:1]: Is this file really needed or is it a leftover?

File: scripts/bump-versions-and-publish-npm.sh
Comment (sergeytimoshin) [inline scripts/bump-versions-and-publish-npm.sh:1]: btw have you seen `nx release` action? https://nx.dev/recipes/nx-release/publish-in-ci-cd

**Review by sergeytimoshin — CHANGES_REQUESTED** [review]

File: .npmrc.example (hunk 1 above)
Comment (SwenSchaeferjohann) [reply .npmrc.example:1]: yes it's useful because we should not commit the actual `.npmrc` file

File: scripts/bump-versions-and-publish-npm.sh
Comment (SwenSchaeferjohann) [reply scripts/bump-versions-and-publish-npm.sh:1]: thanks for sharing, let's consider switching to that in a follow up pr when we actually integrate these into CI

File: .npmrc.example (hunk 1 above)
Comment (vadorovsky) [reply .npmrc.example:1]: I think it can be useful, but can we also add `.npmrc` to gitignore?

**Review by vadorovsky — APPROVED** [review]
Looks good, we just need to resolve the CI errors

File: .npmrc.example (hunk 1 above)
Comment (SwenSchaeferjohann) [reply .npmrc.example:1]: yup already is

**Review by vadorovsky — APPROVED** [review]

**Review by sergeytimoshin — APPROVED** [review]
